### DISCUSSION TITLE
ScriptReference/Object.DestroyImmediate

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#44] [Notes]
# ScriptReference/Object.DestroyImmediate

https://docs.unity3d.com/ScriptReference/Object.DestroyImmediate.html

**Comment by vertxxyz** [comment]
When destroying prefab assets you must use [`AssetDatabase.DeleteAsset(path);`](https://docs.unity3d.com/ScriptReference/AssetDatabase.DeleteAsset.html) as `Object.DestroyImmediate(prefabAsset, true);` will not successfully delete the asset.
